src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/treasury/opi/wfingestion/TreasuryOpiIngestionWFImpl.java

```diff
@@ -55,19 +55,12 @@ public void ingest(Long ingestionFlowFileId) {
       ingestionResult.getErrorDescription(),
       ingestionResult.getDiscardedFileName());
 
-    ingestionResult.getIufs().forEach(iuf -> {
-
-      // TODO P4ADEV-1936 replace fake values with real ones for organizationId and treasuryId
-      // ingestionResult.getOrganizationId()
-      // ingestionResult.getTreasuryId()
-
+    ingestionResult.getIufTreasuryIdMap().forEach((key, value) ->
```

Comment:
let's remove the comment below to give a name to key and value parameter, just give them the right value
```suggestion
    ingestionResult.getIufTreasuryIdMap().forEach((iuf, treasuryId) ->
```